test-suites.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
```

Comment:
This should probably call the test-gui script, shouldn't it?
But do we need to run multiple suites? I thought that the idea was to add a new job running single suite.

Reply:
we don't run multiple suites - we need to run only kubevirt-gating suite in addition to the other tests that run regularly in each PR.

Reply:
done.